### DISCUSSION TITLE
Great Firmware

### DISCUSSION THREAD

**Discussion by PJMCL1618033** [OpenNeptune3D/OpenNept4une#143] [General]
I installed this firmware on a Neptune 4 Pro and want to say it is very good. I like how the attached display shows the model thumbnail and also assists in tramming the print plate. I also like how easy it is to update firmware and the adaptive mesh. This firmware turned a OK 3D printer into a very good one. The Wiki help section is very informative for an easy install and setup. 

I had specifically purchased a Neptune 4 because it was mostly an open source machine and had the hope it could be transformed into a completely open source one. Your firmware did exactly that. Where can I contribute monetarily to this project?

Thank you.

**Comment by halfmanbear** [comment]
Thankyou for your feedback! I'm glad it's working well for you.

We have our sponsor links on the main GitHub page for the project
